Kitodo/src/main/java/org/kitodo/production/model/bibliography/course/metadata/CountableMetadata.java

```diff
@@ -259,9 +269,29 @@ public void setStepSize(Granularity stepSize) {
      *            the start value to set
      */
     public void setStartValue(String startValue) {
+        ((ProcessTextMetadata)metadataDetail).setValue(startValue);
         this.startValue = startValue;
     }
 
+    /**
+     * Gets metadataDetail.
+     *
+     * @return value of metadataDetail
+     */
+    public ProcessDetail getMetadataDetail() {
+        return metadataDetail;
+    }
+
+    /**
+     * Sets metadataDetail.
+     *
+     * @param metadataDetail value of metadataDetail
+     */
+    public void setMetadataDetail(ProcessDetail metadataDetail) {
+        this.metadataDetail = metadataDetail;
+        this.metadataType = this.metadataDetail.getLabel();
```

Comment:
Doesn't `getLabel` return a value translated to the language that is currently selected? So that wouldn't work as a "type", would it?

Reply:
Yes, I correct it. It should be `this.metadataType = this.metadataDetail.getMetadataID();`